src/patternfly/components/AppLauncher/examples/app-launcher-sections-icons-example.hbs

```diff
@@ -0,0 +1,266 @@
+{{#> app-launcher id="app-launcher-example-sections-icons" app-launcher--IsExpanded="true" aria-label="Actions" app-launcher--IsNav="true" app-launcher--attribute='aria-label="global"'}}
+  {{#> app-launcher-group}}
+    <ul>
+      <li>
+        {{#> app-launcher-menu-item}}
+          {{#> app-launcher-menu-item-icon}}
+            <img src="https://kubernetes.io/images/favicon.png" alt="">
```

Comment:
> Should aria-hidden="true" be applied to the icon on the left of the list item, too?

Since you are using `<img>` elements for these icons, AND since these images are redundant (i.e. the text that follows is considered synonymous with these icons), then having `alt=""` like you have it here is the correct way to tell assistive technology that this image provides no extra meaning and to ignore it.